SECURITY.md

```diff
@@ -0,0 +1,110 @@
+# Security Reporting Process
+
+Please report any security issue or OpenTofu crash report to
```

Comment:
Do we really want crash reports at this email alias? Seems like crash reports should be filed as bugs in the repo, and only vulnerability disclosures should go to the email address?